### PR TITLE
Adds Silvia the snake, a pet for CMO

### PR DESCRIPTION
**Pull request by IamVelcroboy**
## About the PR
Silvia the snake is a space cobra with omnizine venom, meaning she can heal with her bites.
She has a limited amount of venom that regens slowly(15 mins to full regen)
Full regen can heal a person to 90hp(burn/brute/air/toxin)
She has the same, slightly nerfed, abilities as other space cobras, meaning she can go invisible and glows a little bit.

## Why / Balance
Because rad!

## Technical details
- Someone please make it so she can wear a nurse hat. I tried but because she moves it's basically impossible to get them to time up together. 🤷‍♂️ 
- ~~Maybe add an additional healing item when butchered later~~
- Maybe add special snake boots later
- Needs a snake accent for when cognizined!

## Media
image
image
image
image
image

## Breaking changes

**Changelog**
:cl: Velcroboy
- add: Added Silvia the snake, a pet with healing venom as CMO's new pet

### CODE REVIEW

**Comment by Bonktrauma** [comment]
This is adorable,  but two questions, Can they be milked?(like normal snakes) either by biting a beaker, or something else, 

And Secondly, is it a ghost role?

**Comment by IamVelcroboy** [comment]
> This is adorable, but two questions, Can they be milked?(like normal snakes) either by biting a beaker, or something else,

That's like, the one thing I didn't test. But... Probably. If so it could be pretty OP and might need to be diluted or something. 

> And Secondly, is it a ghost role?

Yes, Definitely

**Comment by Bonktrauma** [comment]
> > This is adorable, but two questions, Can they be milked?(like normal snakes) either by biting a beaker, or something else,
> 
> That's like, the one thing I didn't test. But... Probably. If so it could be pretty OP and might need to be diluted or something. 
> 
> > And Secondly, is it a ghost role?
> 
> Yes, Definitely 
> 
> 

Here's an idea, mix the omnizine with a venom that can kill regardless of the omni, "unprocessed venom" and make Chem have to separate the two so they can actually be able to fight off attackers AND it can't just heal anybody

**Comment by IamVelcroboy** [comment]
> Here's an idea, mix the omnizine with a venom that can kill regardless of the omni, "unprocessed venom" and make Chem have to separate the two so they can actually be able to fight off attackers AND it can't just heal anybody

Did a little testing and as far as I can tell, extracting venom with a beaker isn't a thing you can do currently.  Unless I just don't know how?

**Comment by Bonktrauma** [comment]
> > Here's an idea, mix the omnizine with a venom that can kill regardless of the omni, "unprocessed venom" and make Chem have to separate the two so they can actually be able to fight off attackers AND it can't just heal anybody
> 
> Did a little testing and as far as I can tell, extracting venom with a beaker isn't a thing you can do currently.  Unless I just don't know how?

I see

File: Resources/Prototypes/DeltaV/Objectives/stealTargetGroups.yml [hunk 1]
```diff
@@ -26,4 +26,9 @@
     sprite: DeltaV/Objects/Misc/stamps.rsi
     state: stamp-notary
 
-
+- type: stealTargetGroup
+  id: AnimalSilvia
+  name: Shiva
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Objectives/stealTargetGroups.yml]:
```suggestion
  name: Silvia
```
:trollface:

**Review by Colin-Tel — COMMENTED** [review]
Wouldn't adding this pet as a steal objective mean that we'd need to map them in all our stations?

File: Resources/Prototypes/DeltaV/Objectives/stealTargetGroups.yml (hunk 1 above)
Comment (IamVelcroboy) [reply Resources/Prototypes/DeltaV/Objectives/stealTargetGroups.yml]: 😋

**Comment by IamVelcroboy** [comment]
Adding to steal, would I suppose, require each station to have one. Is that an issue? (I sure do wish the event itself checked for entities before assigning them but alas...)

**Comment by IamVelcroboy** [comment]
Shiva is also a steal and some of our maps use Laika or random sec animal spawner....I suppose this is also an issue.

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/pets.yml [hunk 2]
```diff
@@ -76,3 +76,132 @@
...
+          Base: silvia
+          BaseUnshaded: glow
+        Dead:
+          Base: dead_silvia
+    - type: Butcherable
+      spawned:
+        - id: FoodMeatSnake
+          amount: 2
+        - id: OmnizineChemistryBottle
+          amount: 1
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/pets.yml]: should be a special omni-rich meat not you butcher snake and get a glass bottle out of thin air

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/pets.yml (hunk 2 above)
Comment (IamVelcroboy) [reply Resources/Prototypes/DeltaV/Entities/Mobs/NPCs/pets.yml]: Added a meat

**Comment by IamVelcroboy** [comment]
> Wouldn't adding this pet as a steal objective mean that we'd need to map them in all our stations?

Is there any further concern with this?

**Review by deltanedas — APPROVED** [review]

**Comment by IamVelcroboy** [comment]
How does this not have a conflict?

**Comment by IamVelcroboy** [comment]
ok well...

**Pull request merged**